### DISCUSSION TITLE
Proposal for Updating and Verfying Template Scaffolding Instances

### DISCUSSION THREAD

**Discussion by msclock** [serious-scaffold/.github#3] [Ideas]
The proposal aims to discuss how to embed a mantainable solution to scaffold templates for updating and verifying scaffolding instances. 

Initially, we can use a munual solution from copier. Stilly,we should have a mechanism to infom user endpoints.

Here I simply recommend using existing dependency management tools and ci checks, such as
- renovate combined with github action.
- renovate combined with gitlab ci.

Any other solutions are welcome and appreciated. Thank you for your time and disucssion.

**Comment by huxuan** [comment]
It looks good to me, it can be part of the scaffold so that users can receive automatically generated pull/merge request.

**Comment by huxuan** [comment]
BTW, there is a discussion that refers to https://github.com/KyleKing/copier-template-tester that we can use to test the scaffold ourselves.

https://github.com/copier-org/copier/issues/1384#issuecomment-1783673586
